v23.1/transaction-retry-error-reference.md

```diff
@@ -1,54 +1,92 @@
 ---
 title: Transaction Retry Error Reference
-summary: A list of the transaction retry (serialization) errors emitted by CockroachDB, including likely causes and user actions for mitigation.
+summary: A list of the transaction retry errors emitted by CockroachDB, including likely causes and user actions for mitigation.
 toc: true
 docs_area: reference.transaction_retry_error_reference
 ---
 
-This page has a list of the transaction retry error codes emitted by CockroachDB.
+When a [transaction](transactions.html) is unable to complete due to another concurrent or recent transaction attempting to write to the same data, CockroachDB will [automatically attempt to retry the failed transaction](transactions.html#automatic-retries) without involving the client (i.e., silently). If the automatic retry is not possible or fails, a _transaction retry error_ is emitted to the client. Multiple transactions trying to write to the same data is also known as contention.
 
-Transaction retry errors use the `SQLSTATE` error code `40001`, and emit error messages with the string `restart transaction`. The most common transaction retry errors are also known as serialization errors. Serialization errors indicate that a transaction failed because it could not be placed into a [serializable ordering](demo-serializable.html) among all of the currently-executing transactions.
+Transaction retry errors fall into two categories:
 
-The failure to place a transaction into a serializable ordering usually happens due to a [transaction conflict](architecture/transaction-layer.html#transaction-conflicts) with another concurrent or recent transaction that is trying to write to the same data. When multiple transactions are trying to write to the same data, this state is also known as contention. When serialization errors occur due to contention, the transaction needs to be retried by the client as described in [client-side retry handling](transactions.html#client-side-intervention).
+- **Serialization Errors** indicate that a transaction failed because it could not be placed into a [serializable ordering](demo-serializable.html) among all of the currently-executing transactions. These errors are generally addressed with client-side intervention, where the client [initiates a restart of the transaction](#client-side-retry-handling), and [adjusts application logic and tunes queries](#minimize-transaction-retry-errors) for greater performance.
```

Comment:
Not 100% sure as this isn't my area, but this wording sounds incorrect. "Could not be placed into a serializable ordering" seems garbled/redundant, because the 'series' and 'order' are the same concept. So you could say that the transaction 'could not be serialized' or 'could not be placed into an (valid? determinate?) ordering', but combining into 'serializable ordering' seems problematic. Although maybe I just don't get it, totes possible